benchmarking/benchmark.py

```diff
@@ -0,0 +1,346 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+from scipy.sparse import csr_array
+
+import rdkit.Chem.rdFingerprintGenerator as fpgens
+from rdkit.Chem.rdMolDescriptors import GetMACCSKeysFingerprint
+from rdkit.Chem.rdReducedGraphs import GetErGFingerprint
+
+from time import time
+
+from featurizers.fingerprints import (
+    MorganFingerprint,
+    AtomPairFingerprint,
+    TopologicalTorsionFingerprint,
+    MACCSKeysFingerprint,
+    ERGFingerprint,
+)
```

Comment:
Use black + isort for import sorting in notebook too for proper PEP8 order. Don't we have a pre-commit hook for this?

Reply:
Add separate Poetry dependencies environment in `pyproject.toml` for benchmarking. You can have multiple sections in TOML file, e.g. main, dev, test etc., this is useful in cases like this. Alternatively, you can make a separate file, or even just requirements.txt.

Reply:
This code probably relies on PyTorch. I suggest adding PyTorch CPU-only to benchmark dependencies like this: https://stackoverflow.com/a/76359919/9472066.

Reply:
Isort and black format done. Same with python dependencies. I'm not sure what to do with .toml dependencies as I don't have much experience with that. I might be able to resolve it later